packages/contracts-bedrock/scripts/deploy/Deploy.s.sol

```diff
@@ -823,7 +827,8 @@ contract Deploy is Deployer {
             _data: abi.encodeCall(
                 ISystemConfig.initialize,
                 (
```

Comment:
I think it should be a new config value at this layer of abstraction and `op-deployer` can handle setting it to the correct thing. For new chains in the superchain, it should not be the same value as the owner of the system config but some chains may want it to be, so by adding a new config value it allows for flexibility while at a higher abstraction the `useStandard` config can make sure the correct value is set